### PR TITLE
Changed subchunk request origin to match vanilla

### PR DESCRIPTION
**Pull request by FreezeEngine**

### CODE REVIEW

**Comment by extremeheat** [comment]
This doesn't seem right. Are you sure this is correct? The tests are passing on master, failing here.

**Comment by FreezeEngine** [comment]
> This doesn't seem right. Are you sure this is correct? The tests are passing on master, failing here.

I only changed 1 number, on my machine tests are passing, I havn't tried to run it again because it would be strange if it failed one time and worked second time

File: test/makeFlatWorld.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ describe('make flat world', function () {
   for (const version of versions) {
     const registry = Registry('bedrock_' + version)
     const worldPath = join(__dirname, '/flat-world-' + version)
-    const World = PrismarineWorld(registry)
+    const World = PrismarineWorld('bedrock_' + version)
```
Comment (extremeheat) [inline test/makeFlatWorld.ts:18]: This is not a correct change

File: test/makeFlatWorld.ts [hunk 2]
```diff
@@ -47,7 +47,6 @@ describe('make flat world', function () {
       world.setLoadedColumn(0, 0, cc)
       world.queueSaving(0, 0)
       await world.saveNow() // Why does this not return proper Promise?
-      await world.finishedSaving
```
Comment (extremeheat) [inline test/makeFlatWorld.ts:50]: Why?

**Review by extremeheat — CHANGES_REQUESTED** [review]

File: test/makeFlatWorld.ts (hunk 2 above)
Comment (FreezeEngine) [reply test/makeFlatWorld.ts:50]: Its private in new pworld

File: test/makeFlatWorld.ts (hunk 1 above)
Comment (FreezeEngine) [reply test/makeFlatWorld.ts:18]: It requires string, not registry, ts compile was failing

Comment (extremeheat) [reply test/makeFlatWorld.ts:18]: Ah, then this needs to be fixed in primarine-world

**Comment by rom1504** [comment]
please fix in prismarine-world

**Pull request closed without merging**